circle.yml

```diff
@@ -8,7 +8,14 @@ machine:
 
 ## Customize deployment commands
 deployment:
-  staging:
+  production:
+    branch: master
+    commands:
+      - heroku pg:backups  --app vote-kompot
+      - git push git@heroku.com:vote-kompot.git $CIRCLE_SHA1:refs/heads/master
+      - heroku run rake db:migrate --app vote-kompot
+deployment:
```

Comment:
I try to do it in one block and get exception evry time. Don't know why, this is only way that I figure out to do. Official documentation and examples shows only one change in block, any glue how to do heroku and commands configuration either. But previos time I just place it after each other and it works, no idea what different now.